main_test.go

```diff
@@ -53,6 +61,29 @@ func TestSendMetrics(t *testing.T) {
 	assert.NoError(err)
 }
 
+func TestSendMetricsHostStripping(t *testing.T) {
+	for _, tt := range hostStrippingDataSet {
```

Comment:
Once you've moved the test cases into the test function itself, you can also add the field `stripHost`. This will allow you to define additional test cases for when `stripHost: false`.
```suggestion
	hostStrippingDataSet := []struct {
		entityName   string
		expectedBody string
		stripHost    bool
	}{
		{
			entityName:   "sensu.company.com",
			expectedBody: `answer,foo=bar,sensu_entity_name=sensu.company.com value=42`,
			stripHost:    true,
		},
		{
			entityName:   "answer.company.com",
			expectedBody: `answer,foo=bar,sensu_entity_name=answer.company.com value=42`,
			stripHost:    true,
		},
	}
	
	for _, tt := range hostStrippingDataSet {
```